saleor/graphql/schema.graphql

```diff
@@ -5261,6 +5267,193 @@ type CategoryCountableEdge {
   cursor: String!
 }
 
+input ProductFilterInput {
```

Comment:
This input type should be already defined in the schema, and this seems like a duplication, isn't it? Did you change this file by hand? To update it you can use a command `npm run build-schema`.

Reply:
Sorry for closing because I failed fix cla not able to change email address in old commits